src/test/java/com/google/sps/integration/WelcomePageIT.java

```diff
@@ -0,0 +1,76 @@
+package com.google.sps.integration;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WelcomePageIT {
+  private WebDriver driver;
+  private final int WAIT_TIME = 30;
+  private final int MAX_NUM_TABS = 2;
+
+  @BeforeClass
+  public static void setUp() {
+    System.setProperty("webdriver.chrome.driver", "resources/chromedriver");
+  }
+
+  @Before
+  public void testSetUp() {
+    driver = new ChromeDriver();
+  }
+
+  /**
+   * Testing if clicking on TurboVotes button opens a new tab and redirects the user to the
+   * TurboVotes page.
+   */
+  @Test
+  public void testTurboVotesButton() {
+    driver.get("http://localhost:9876");
+
+    // Ensure only one window is open.
```

Comment:
These comments are not necessary since it's describing _what_ is happening as opposed to _why_.  Generally the code should be already readable where it's easy to tell what's going on by inspection.  There are exceptions to this, but pretty rare.